src/config.rs

```diff
@@ -444,3 +442,65 @@ enabled = true
     }
     Ok(())
 }
+
+pub fn remove_forge_lib() -> Result<(), ConfigError> {
+    let lib_dir = get_current_working_dir().unwrap().join("lib/");
+    let gitmodules_file = get_current_working_dir().unwrap().join(".gitmodules");
+    if !gitmodules_file.exists() {
+        return Err(ConfigError{
+            cause: "Couldn't find .gitmodules".to_string()
+        })
+    }
+    if !lib_dir.exists() {
+        return Err(ConfigError{
+            cause: "Couldn't find the lib directory".to_string()
+        }) 
+    }
+    match remove_file(gitmodules_file) {
+        Ok(_) => {}
+        Err(err) => {
+            return Err(ConfigError {
+                cause: format!("Couldn't remove the .gitmodules file: {}", err),
+            });
+        }
+    }
+    match remove_dir_all(lib_dir) {
+        Ok(_) => {}
+        Err(err) => {
+            return Err(ConfigError {
+                cause: format!("Couldn't remove the lib dirrectory: {}", err),
+            });
+        }
+    }
+    Ok(())
+}
+
+pub fn create_default_config_file(forge_version: &str) -> Result<(), ConfigError> {
+    let path: PathBuf = get_current_working_dir().unwrap().join("foundry.toml");
+    let config_file = path.to_str().unwrap();
+
+    let content:&str = &format!(r#"
+[profile.default]
```

Comment:
hmm, is this deleting the whole foundry file and creating a new one? cause I would just parse the foundry file and add the forge-std to the dependencies I wouldn't create one.
We should first check if the config file exists and if that config file is foundry or not. if it's not foundry we just stop this init cause this init is just for foundry.toml configs not for soldeer.toml. 
We support soldeer.toml also for projects that might not use foundry as their dep manager.
For remappings, for now we add everything in the `remappings.txt` file, there is a function that adds to remappings. 
Once we ready we move the logic of that function to add to the foundry=>remappings field.

Reply:
Okay, the reason I deleted and created it again is because its supposed to be used after initializing a forge project, and it changes `libs = ["lib"]` to `libs = ["dependencies"]`

Reply:
Understood but we can't rely on what info we will actually replace in the initial config.
Maybe just process the toml and see if libs is defined, if so then just replace with `dependencies` if not just add this `Item` in the toml

Reply:
also if libs is defined, just replace the `"lib"` and not everything as the user might have defined multiple targets